microsoft-edge/web-platform/ad-selection-api.md

```diff
@@ -0,0 +1,282 @@
+
+   * In the **Subdomain Support** section, select the **Yes** or **No** option button.
+
+   * In the **Email Address** text box, provide a valid developer contact for the domain.
+
+   * In the **Origin Trial Name** text box, enter **Ad Selection API**.
+
+1. Click the **Submit** button.
+
+   After you receive an OT token for the domain, use the OT token in the `attestations.json` file for that domain, to enable testing the Ad Selection API with supported Microsoft Edge clients.
+
+1. To complete your attestation and allow for continued access to the Ad Selection API, create a JSON file named `attestations.json` and make it available at the `/.well-known/attestations.json` path for each registered domain.  For example: `https://contoso.example/.well-known/ad-selection-attestations.json`.
+
+   The `attestations.json` file for a domain must conform to the following standards below, and must include the created OT token as well as a list of APIs.
+
+
+<!-- ------------------------------ -->
+#### Example attestations.json file
+
+Below is an example of an `attestations.json` JSON file, containing an OT token, which must be published within **30 days** of receiving the initial token:
+
+```json
+{
+    "ad_selection_api_attestations": [
+        {
+            "attestation_parser_version": "2",
+            "attestation_version": "2",
+            "privacy_policy": [
+                "https://contoso.example/privacy/"
+            ],
+            "ownership_token": "<Your Unique Token Here>",
+            "enrollment_site": "https://contoso.example/",
+            "platform_attestations": [
+                {
+                    "platform": "edge",
+                    "attestations": {
+                        "attribution_reporting_api": {
+                            "ServiceNotUsedForIdentifyingUserAcrossSites": true/false
+                        },
+                        "shared_storage_api": {
+                            "ServiceNotUsedForIdentifyingUserAcrossSites": true/false
+                        },
+                        "private_aggregation_api": {
+                            "ServiceNotUsedForIdentifyingUserAcrossSites": true/false
+                        },
+                        "ad_selection_api": {
+                            "ServiceNotUsedForIdentifyingUserAcrossSites": true/false
+                        }
+                    }
+                },
+                {
+                    "platform": "android",
+                    "attestations": {}
+                }
+            ]
+        }
+    ]
+}
+```
+
+
+<!-- ---------- -->
+###### Important fields and values
+
+* `"ownership_token":` is the OT token that was generated when registering this domain for the **Ad Selection API** origin trial.
+
+* In the `"platform_attestations":` section, `"platform":` must be `"edge"` or `"android"`.
+
+* In the `"platform_attestations":` section, valid members of `"attestations":` are the following APIs:
```

Comment:
Should this article say anything about these four APIs of the Ad Selection API, that are apparently enabled by the OT token?
* attribution_reporting_api
* shared_storage_api
* private_aggregation_api
* ad_selection_api

according to wording in json, the Ad Selection API consists of the following APIs (one of which is ad_selection_api):

The OT token enables using these four APIs of the API:
* attribution_reporting_api
* shared_storage_api
* private_aggregation_api
* ad_selection_api

Find "attribution" and "reporting", to see if this API is discussed: no.
Find "storage", to see if this API is discussed: no.
Find "aggregat", to see if this API is discussed: no.

draft Doc:
![image](https://github.com/user-attachments/assets/6ca010c7-f917-41f4-a5d6-1dccc429abd7)
fyi @captainbrosset

Reply:
clarified w/ draft text in meeting